components/ModEditSubmissionForm.vue

```diff
@@ -515,15 +519,22 @@ import { validateAddressLineEn,
     validateFloat,
     validatePostalCode,
     validateWebsite,
-    validateCityJa,
-    validateUserSubmittedFirstName,
-    validateUserSubmittedLastName } from '~/utils/formValidations'
+    validateCityJa } from '~/utils/formValidations'
 import { ModSubmissionLeftNavbarSectionIDs, useModerationScreenStore, ModerationScreen } from '~/stores/moderationScreenStore'
+import { onBeforeRouteLeave } from '#app'
+import { useI18n } from '#imports'
 import { multiSelectWithoutKeyboard } from '~/utils/multiSelectWithoutKeyboard'
 import { useModalStore } from '~/stores/modalStore'
-import { onBeforeRouteLeave } from '#app'
+import SVGTrashCan from '~/assets/icons/trash-can.svg'
+import SVGProfileIcon from '~/assets/icons/profile-icon.svg'
+import { triggerFormValidationErrorMessages } from '~/utils/triggerFormValidationErrorMessages'
+
+let toast: ToastInterface
```

Comment:
Is there a reason this isn't a `const` assigned to `useToast()`?

Reply:
This is the thing I have been talking about with documentation. I will add comments. But if you run it before onMounted it crashes the build cuz it is only a user component. You have to do it `onMounted` or everything will fail due to the way Nuxt builds with the SSR and setup. It won't be able to find `useToast`

Reply:
just wrap it in a client side check and it should work fine?

Reply:
It hasn't with Nuxt but I will try

Reply:
@ermish  This is the error I get if I try without doing it onMounted

![image](https://github.com/user-attachments/assets/d6eb9fee-a649-4c07-a6f2-6d001f65bdd1)